Formula/neovim.rb

```diff
@@ -18,7 +18,7 @@ class Neovim < Formula
     end
 
     resource "luajit" do
-      url "http://luajit.org/download/LuaJIT-2.0.4.tar.gz"
+      url "https://github.com/neovim/deps/blob/master/opt/LuaJIT-2.0.4.tar.gz"
```

Comment:
*facepalm* I'm sorry, I don't know how that happened. I did test the formula but it must have been using a cached copy. I changed the URL and checked that it's good.